auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -5,15 +5,14 @@
 
 from openerp import models, api, tools, SUPERUSER_ID
 
-
 DELAY_KEY = 'inactive_session_time_out_delay'
 IGNORED_PATH_KEY = 'inactive_session_time_out_ignored_url'
 
 
 class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
-    @tools.ormcache(skiparg=0)
+    @tools.ormcache('db')
     def get_session_parameters(self, db):
```

Comment:
I followed @yajo's advice here (https://github.com/OCA/server-tools/pull/875#discussion_r124193147) and got rid of this whole file in favour of new api and method improvements (wich seem to be available in v9 as well: https://github.com/odoo/odoo/blob/9.0/openerp/addons/base/ir/ir_config_parameter.py#L57).